x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.test.tsx

```diff
@@ -85,6 +85,12 @@ describe('When the add exception modal is opened', () => {
     }));
   });
 
+  const alertDataMock: AlertData = {
```

Comment:
Is `AlertData` mutable? Since this is allocated just once yet used for several tests, a test could mutate it and that would effect the other tests.